lib/index.ts

```diff
@@ -76,7 +76,7 @@ const sdk = (...args: Parameters<typeof createSdk>) => {
       // Do not hard-code the algo because library does not support `None` so all are valid
       const res = await jwtVerify(token, this.getKey, { issuer: projectId, clockTolerance: 5 })
 
-      return { token: res.payload }
+      return { token: res.payload, jwt: token }
```

Comment:
Why do we need to return the jwt? 
this is something we get from the fn params, no?

Reply:
But this change is in the `validateToken` fn
What am I missing?